src/map.cpp

```diff
@@ -122,43 +122,35 @@ MapSector * Map::getSectorNoGenerate(v2s16 p)
 	return getSectorNoGenerateNoLock(p);
 }
 
-MapBlock * Map::getBlockNoCreateNoEx(v3s16 p3d)
+MapBlock * Map::getBlockNoCreate(v3s16 p3d)
 {
 	v2s16 p2d(p3d.X, p3d.Z);
 	MapSector * sector = getSectorNoGenerate(p2d);
```

Comment:
```suggestion
	MapSector *sector = getSectorNoGenerate(p2d);
```

PR doesn't touch this line, but it'd be great if this minor nit could be fixed.